plugins/kubernetes/app/assets/javascripts/kubernetes/filters/append_filter.js

```diff
@@ -0,0 +1,7 @@
+samson.filter('append', function(){
+
+  return function(input, str) {
+    return _.isNotUndefinedOrEmpty(input) ? input + str : input;
```

Comment:
Hum, I don't want to change the input when the input is undefined. The idea behind this filter is just to append a given input value with the provided string suffix when the input is defined, otherwise, return the original input. If input is undefined, it should be ignored, otherwise, we will be changing the original input, which can have impact on another filter when when chaining multiple filters.